cvpy/base/ImageTable.py

```diff
@@ -290,13 +290,18 @@ def load(connection: CAS, path: str, load_parms: Dict[str, str] = None,
         # Create a cas table
         cas_table = connection.CASTable(**output_table_parms)
 
+        # Get user specified image_type
+        image_type = None
```

Comment:
Thanks for updating this, I remember this was giving me an error yesterday, but I was able to fi the unit test.